Gemfile.lock

```diff
@@ -43,83 +43,82 @@ GEM
     coffee-rails (3.2.2)
       coffee-script (>= 2.2.0)
       railties (~> 3.2.0)
-    coffee-script (2.3.0)
+    coffee-script (2.4.1)
       coffee-script-source
       execjs
-    coffee-script-source (1.7.1)
-    cucumber (1.3.16)
+    coffee-script-source (1.9.1.1)
+    cucumber (2.1.0)
       builder (>= 2.1.2)
+      cucumber-core (~> 1.3.0)
       diff-lcs (>= 1.1.3)
-      gherkin (~> 2.12)
+      gherkin3 (~> 3.1.0)
       multi_json (>= 1.7.5, < 2.0)
-      multi_test (>= 0.1.1)
-    cucumber-rails (1.4.1)
-      capybara (>= 1.1.2, < 3)
-      cucumber (>= 1.3.8, < 2)
-      mime-types (~> 1.16)
-      nokogiri (~> 1.5)
-      rails (>= 3, < 5)
-    database_cleaner (1.3.0)
-    diff-lcs (1.2.5)
+      multi_test (>= 0.1.2)
+    cucumber-core (1.3.0)
+      gherkin3 (~> 3.1.0)
+    cucumber-rails (1.3.0)
+      capybara (>= 1.1.2)
+      cucumber (>= 1.1.8)
+      nokogiri (>= 1.5.0)
+    database_cleaner (0.9.1)
+    diff-lcs (1.1.3)
     erubis (2.7.0)
-    execjs (2.2.1)
-    faraday (0.9.0)
+    execjs (2.6.0)
+    faraday (0.9.1)
       multipart-post (>= 1.2, < 3)
-    gherkin (2.12.2)
-      multi_json (~> 1.3)
-    haml (4.0.5)
-      tilt
-    hashie (3.2.0)
+    gherkin3 (3.1.1)
+    haml (3.1.7)
+    hashie (1.2.0)
     hike (1.2.3)
-    httparty (0.13.1)
-      json (~> 1.8)
-      multi_xml (>= 0.5.2)
-    i18n (0.6.11)
+    httparty (0.10.0)
```

Comment:
Nice catch; I think I mixed it up for the global version on my localhost as this repository wasn't using rvm before. Will revert to its original version.